programs/config_api/src/config_processor.rs

```diff
@@ -5,6 +5,7 @@ use bincode::deserialize;
 use log::*;
 use solana_sdk::account::KeyedAccount;
 use solana_sdk::instruction::InstructionError;
+use solana_sdk::instruction_processor_utils::next_keyed_account;
```

Comment:
@garious Why isn't this in `solana_sdk::account`?

Reply:
Because of its dependency on InstructionError

Reply:
@garious How about we split `InstructionError` our of `Instruction.rs` into`instruction_error.rs` and have `account.rs` depend on `instruction_error.rs`?

Reply:
Why bother?  I'm not a fan of error modules. See result.rs in `solana_core` for an example of how badly that can go.

Reply:
Currently the function is only used by instruction processors, so being in instruction_processor_utils seems like a fine place to land.

Reply:
Ok, sounds good